src/test/ui/impl-trait/must_outlive_least_region_or_bound.nll.stderr

```diff
@@ -26,7 +26,34 @@ LL | fn explicit<'a>(x: &'a i32) -> impl Copy + 'a { x }
    |                                ^^^^^^^^^^^^^^
 
 error: lifetime may not live long enough
-  --> $DIR/must_outlive_least_region_or_bound.rs:12:69
+  --> $DIR/must_outlive_least_region_or_bound.rs:9:46
+   |
+LL | fn elided2(x: &i32) -> impl Copy + 'static { x }
+   |               -                              ^ returning this value requires that `'1` must outlive `'static`
+   |               |
+   |               let's call the lifetime of this reference `'1`
+   |
+   = help: consider replacing `'1` with `'static`
```

Comment:
Technically, `'1` doesn't appear in the code, so it can't be replaced. Can we suggest changing the type of `x` instead?

Reply:
These are the currently existing nll errors, which I didn't modify one way or another.